.github/workflows/static_analysis.yml

```diff
@@ -20,17 +20,13 @@ jobs:
     - name: Cleanup repository
       run: rm -rf vendor/
 
-    - name: Vet
-      run: go vet -tags ci ./...
-
-    - name: Goimports
-      run: test -z $(goimports -e -d . | tee /dev/stderr)
-
-    - name: Gocyclo
-      run: gocyclo -over 50 .
-
-    - name: Golint
-      run: golint -set_exit_status $(go list -tags ci ./...)
```

Comment:
You can remove the installation of these tools (except for staticcheck) further up in the file.

Reply:
right, I forgot.